docs/source/tutorials/HLS/HLS_search.ipynb

```diff
@@ -0,0 +1,831 @@
+    "    creds = maap.aws.earthdata_s3_credentials('https://data.lpdaac.earthdatacloud.nasa.gov/s3credentials')\n",
+    "    aws_session = boto3.session.Session(\n",
+    "        aws_access_key_id=creds['accessKeyId'], \n",
+    "        aws_secret_access_key=creds['secretAccessKey'],\n",
+    "        aws_session_token=creds['sessionToken'],\n",
+    "        region_name='us-west-2')\n",
+    "    s3 = aws_session.client('s3')\n",
+    "    \n",
+    "    with open(catalog) as f:\n",
+    "        clean_features = []\n",
+    "        asset_catalog = json.load(f)\n",
+    "   \n",
+    "        # Remove duplicate scenes\n",
+    "        features = asset_catalog['features']\n",
+    "        \n",
+    "        for feature in features:\n",
+    "            umm_json = feature['links'][6]['href']\n",
+    "            umm_data = !curl -i {umm_json}\n",
+    "            for line in umm_data:\n",
+    "                if \"CLOUD_COVERAGE\" in line:\n",
+    "                    cc_perc = (int(line.split(\"CLOUD_COVERAGE\")[-1].split('\"')[4]))\n",
+    "                    if cc_perc > cloud_cover:\n",
+    "                        pass\n",
+    "                    else:\n",
+    "                        try:\n",
+    "                            for band in bands:\n",
+    "                                output_file = feature['assets'][band]['href'].replace('https://data.lpdaac.earthdatacloud.nasa.gov/', s3_path)\n",
+    "                                bucket_name = output_file.split(\"/\")[2]\n",
+    "                                s3_key = \"/\".join(output_file.split(\"/\")[3:])\n",
+    "                                head = s3.head_object(Bucket = bucket_name, Key = s3_key, RequestPayer='requester')\n",
+    "                                if head['ResponseMetadata']['HTTPStatusCode'] == 200:\n",
+    "                                    feature['assets'][band]['href'] = output_file\n",
+    "                            clean_features.append(feature)\n",
+    "                        except botocore.exceptions.ClientError as e:\n",
+    "                            if e.response['Error']['Code'] == \"404\":\n",
+    "                                print(f\"The object does not exist. {output_file}\")\n",
+    "                            else:\n",
+    "                                raise\n",
+    "        # save and updated catalog with local paths\n",
+    "        asset_catalog['features'] = clean_features\n",
+    "        local_catalog = catalog.replace('sample', 'local-s3')\n",
+    "        with open(local_catalog,'w') as jsonfile:\n",
+    "            json.dump(asset_catalog, jsonfile)\n",
+    "        \n",
+    "        return local_catalog"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "94ca1a02",
+   "metadata": {},
+   "source": [
+    "#### When run, this will create a new json file which will only include files that meet our cloud cover threshold and have s3 URL's"
```

Comment:
```suggestion
    "When run, this will create a new JSON file that will only include files that meet our cloud cover threshold and have S3 URLs."
```